index.html

```diff
@@ -0,0 +1,157 @@
+      href="image/logo.png">
+   
+      <title>Movie Night</title>
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Archivo:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Actor&family=Klee+One:wght@600&family=Pangolin&display=swap" rel="stylesheet">      
+    </head><body>
+    
+
+    
+
+    <div class="container-nav">
+
+      <div class="nawbar">
+        
+       
+<div class="navbar-items">
+
+
+          <img id="nav-img" src="image/logo.png"/>
+          <h1 id="nav-text">Movie Night</h1>
+
+    
+        
+          <a id="nav-button-1" href="#bignewz" class="nawbar-btn">Veckans Nyhet</a>
+          <a id="nav-button-2" href="#regular-news-section" class="nawbar-btn">Tidigare tips</a>
+          <a id="nav-button-3" href="https://www.aftonbladet.se/tipsa" target="_blank" class="nawbar-btn">Kontakta oss</a>
+          <a id="nav-button-4" href="https://www.aftonbladet.se/nojesbladet/tv" target="_blank" class="nawbar-btn">Utforska</a>
+        </div>
+      </div>
+      </div>
+    
+    
+      <div class="container-header">
+
+      <header>
+        <img src="image/cocaine-cowboys.jpg" alt="cocain cowboys" />
+        <iframe
+          id="video-header"
+          src="https://www.youtube.com/embed/1BN2HF1ZmFI"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </header>
+
+      <section id="bignewz" class="big-news">
+        <h1 id="bignewz-first-h1">
+          VECKANS STREAMINGTIPS:
+          
+          </h1>
+          
+          <h1>”Cocaine cowboys” är Netflix mest dynamiska true
+          crime sedan ”Tiger King”
+        </h1>
+        <p>
+          Netflix svängigaste true crime sedan ”Tiger King”. Billy Corbens
+          sexdelade serie handlar om de Miami-baserade exilkubanerna Sal Magluta
+          och Willy Falcon, som under 80-talets rusiga år lär ha smugglat in och
+          distribuerat över 75 ton colombianskt kokain. Tänk en dokumentär
+          blandning av ”Narcos” och ”Miami vice”; perfekt klippt, spetsad med
+          popkulturella referenser till knarkiga klassiker som ”Gudfadern”,
+          ”Scarface” och ”Pulp fiction”, och med ett ledmotiv där Pitbull rappar
+          ur drogens egen synvinkel. Problematiskt glamouriserande? Kanske.
+          Galopperande underhållande? Tveklöst. Netflix
+        </p> 
+      
+        <a class="stream-big-btn" target="blank" href="https://www.netflix.com/se/title/80996810"
+        >Streama ”Cocaine cowboys” här</a>
+      </section>
+      </div>
+
+
+      <div class="container-regular-footer">
```

Comment:
Thinking of html semantics - is this a footer? Perhaps consider to rename the class?